python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -689,17 +701,40 @@ def quantize(model: nn.Module,
                 model which is able to run on Pytorch or ONNXRuntime can be fetched by
                 `quantized_model.model`.
                 """
-                return inc_quantize(model, inc_calib_dataloader, metric,
-                                    thread_num=thread_num,
-                                    framework=framework,
-                                    conf=conf,
-                                    approach=approach,
-                                    tuning_strategy=tuning_strategy,
-                                    accuracy_criterion=accuracy_criterion,
-                                    timeout=timeout,
-                                    max_trials=max_trials,
-                                    onnxruntime_session_options=onnxruntime_session_options)
 
+                if framework != 'pytorch_ipex':
```

Comment:
We should avoid too much copy-paste

```python
if framework != 'pytorch_ipex':
    try:
        return inc_quantize(...)
    except Exception:
        framework = 'pytorch_ipex'
return PytorchIPEXQuantizationModel(...)
```

Reply:
only 'pytorch_ipex' needs try/except and only 'pytorch_ipex' can use `PytorchIPEXQuantizationModel(...)`

Reply:
Have revised based on the second choice.